episodes/01-introduction.md

```diff
@@ -142,7 +142,7 @@ is covered in
 
 :::::::::::::::::::::::::::::::::::::::: keypoints
 
-- This training aims to teach you a process for designing and developing a lesson website, in collaboration with others.
+- This training aims to teach you a process for designing and developing an online lesson material, in collaboration with others.
```

Comment:
I think I prefer the original wording. Can you elaborate on why you think "online lesson material" is more appropriate?

Reply:
I suppose we are teaching them both - the lesson design (nothing to do with the website) as well as using the Workbench to implement a lesson as a lesson website. 

Happy to receive pushbacks on comments ;-). And happy for this to stay as originally written - will go with your call.

Reply:
ah I see your point now. how about this?

```suggestion
- This training aims to teach you a process for designing a lesson and the skills to develop it as an open source website, in collaboration with others.
```

Reply:
OK, will add that now 👍